src/autoload.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php // phpcs:disable PSR12.Files.FileHeader.IncorrectOrder,PSR12.Files.FileHeader.SpacingAfterBlock,WebimpressCodingStandard.Files.DeclareStrictTypes.BelowComment
```

Comment:
@weierophinney hey, does this fix this issue?

https://github.com/laminas/laminas-coding-standard/blob/1897419d40e2e8b3f66e4f1d2e7cd690b5efd832/src/LaminasCodingStandard/ruleset.xml#L105-L110

Reply:
Likely, but it wasn't present in the version in the lockfile, so the disable rule above takes care of it.